aea/protocols/generator.py

```diff
@@ -585,9 +614,7 @@ def _import_from_custom_types_module(self) -> str:
         :return: import statement for the custom_types module
         """
         import_str = ""
-        if len(self._all_custom_types) == 0:
-            pass
-        else:
+        if self._all_custom_types:
```

Comment:
explicit please

Reply:
you should have a guard clause here: so in case of condition not met, return immediately. See LHS, replace `pass` with `return import_str`

Reply:
Well if the condition in the `if` statement is not met, it just jumps to the next line after `if` which is the `return` and will return the empty string "" assigned to `import_str ` at the beginning of the method.

Reply:
I can read the code. This is about best practice: https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a & https://dev.to/wagslane/guard-clauses-how-to-clean-up-conditionals-2fdm for instance

Reply:
Thanks for the documents. Useful pointer to make the code easy to read. I introduced the guard clause back. 

It is interesting though, looking back, how the other code is easier for me to read and understand than with the one with the guard clause.